### PR TITLE
OSDOCS-4661:adds Agent-based Intaller to the Installing overview page

### PR DESCRIPTION
**Pull request by Amrita42**
- Applies to main and 4.12
- [Jira](https://issues.redhat.com/browse/OSDOCS-4716)
- [Preview](https://54035--docspreview.netlify.app/openshift-enterprise/latest/installing/index.html)
- @celebdor  This PR adds the Agent-based Installer introduction in the Installation overview page. ptal and let me know if i need to add anything. Thanks

### CODE REVIEW

**Comment by Amrita42** [comment]
spacing corrected

File: modules/installation-overview.adoc [hunk 1]
```diff
@@ -22,6 +22,11 @@ Both cluster types have the following characteristics:
 * Highly available infrastructure with no single points of failure is available by default.
 * Administrators maintain control over what updates are applied and when.
 
+The xref:../installing/installing_with_agent_based_installer/preparing-to-install-with-agent-based-installer.adoc#preparing-to-install-with-agent-based-installer[Agent-based Installer]
```
Comment (kcarmichael08) [inline modules/installation-overview.adoc:25]: I think we are not allowed to have xrefs in modules. I think you need to add an "Additional resources" section in the assembly right under where you include this module, and list the link in that, per [these instructions](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#additional-resources-sections).

**Review by kcarmichael08 — COMMENTED** [review]
Please see comment about xref - otherwise LGTM.

File: modules/installation-overview.adoc (hunk 1 above)
Comment (Amrita42) [reply modules/installation-overview.adoc:25]: ooops my mistake , yes you are right. Thanks :)

**Pull request merged**